projects/1.20/assets/project-expansion/projectexpansion/lang/zh_cn.json

```
@@ -0,0 +1,582 @@
+	"advancements.projectexpansion.violet_fuel_block": "紫罗兰燃料块",
+	"advancements.projectexpansion.violet_fuel_block.description": "获得紫罗兰燃料块。",
+	"advancements.projectexpansion.blue_fuel": "蓝燃料",
+	"advancements.projectexpansion.blue_fuel.description": "获得蓝燃料。",
+	"advancements.projectexpansion.blue_fuel_block": "蓝燃料块",
+	"advancements.projectexpansion.blue_fuel_block.description": "获得蓝燃料块。",
+	"advancements.projectexpansion.cyan_fuel": "青燃料",
+	"advancements.projectexpansion.cyan_fuel.description": "获得青燃料。",
+	"advancements.projectexpansion.cyan_fuel_block": "青燃料块",
+	"advancements.projectexpansion.cyan_fuel_block.description": "获得青燃料块。",
+	"advancements.projectexpansion.green_fuel": "绿燃料",
+	"advancements.projectexpansion.green_fuel.description": "获得绿燃料。",
+	"advancements.projectexpansion.green_fuel_block": "绿燃料块",
+	"advancements.projectexpansion.green_fuel_block.description": "获得绿燃料块。",
+	"advancements.projectexpansion.lime_fuel": "黄绿燃料",
+	"advancements.projectexpansion.lime_fuel.description": "获得黄绿燃料。",
+	"advancements.projectexpansion.lime_fuel_block": "黄绿燃料块",
+	"advancements.projectexpansion.lime_fuel_block.description": "获得黄绿燃料块。",
+	"advancements.projectexpansion.yellow_fuel": "黄燃料",
+	"advancements.projectexpansion.yellow_fuel.description": "获得黄燃料。",
+	"advancements.projectexpansion.yellow_fuel_block": "黄燃料块",
+	"advancements.projectexpansion.yellow_fuel_block.description": "获得黄燃料块。",
+	"advancements.projectexpansion.orange_fuel": "橙燃料",
+	"advancements.projectexpansion.orange_fuel.description": "获得橙燃料。",
+	"advancements.projectexpansion.orange_fuel_block": "橙燃料块",
+	"advancements.projectexpansion.orange_fuel_block.description": "获得橙燃料块。",
+	"advancements.projectexpansion.white_fuel": "白燃料",
+	"advancements.projectexpansion.white_fuel.description": "获得白燃料。",
+	"advancements.projectexpansion.white_fuel_block": "白燃料块",
+	"advancements.projectexpansion.white_fuel_block.description": "获得白燃料块。",
+	"advancements.projectexpansion.fading_fuel": "渐消燃料",
+	"advancements.projectexpansion.fading_fuel.description": "获得渐消燃料。",
+	"advancements.projectexpansion.fading_fuel_block": "渐消燃料块",
+	"advancements.projectexpansion.fading_fuel_block.description": "获得渐消燃料块。",
+	"advancements.projectexpansion.magnum_star_omega": "更大的EMC电池？？",
+	"advancements.projectexpansion.magnum_star_omega.description": "应该没有人拥有这么多的能量。",
+	"advancements.projectexpansion.colossal_star_omega": "为什么你需要这么多EMC？",
+	"advancements.projectexpansion.colossal_star_omega.description": "这越来越荒谬了。",
+	"advancements.projectexpansion.final_star_shard": "你玩这个游戏太久了",
+	"advancements.projectexpansion.final_star_shard.description": "说真的，出去走走吧。",
+	"advancements.projectexpansion.transmutation_interface": "电脑里的转化？？",
+	"advancements.projectexpansion.transmutation_interface.description": "它们变得更加出色了。",
+	"advancements.projectexpansion.infinite_steak": "无限牛排！！！",
+	"advancements.projectexpansion.infinite_steak.description": "面包？那是为农民准备的。",
```

```suggestion
	"advancements.projectexpansion.infinite_steak.description": "面包？那是为贫农准备的。",
```
原文peasants，mc里有farmer所以得避讳